insights_sha_extractor_test.sh

```diff
@@ -32,13 +32,12 @@ function clone_service() {
 
 function install_reqs() {
     pip install -r requirements.txt
-    pip install -r requirements/insights_sha_extractor.txt
 }
 
 function prepare_venv() {
     echo "Preparing environment"
     # shellcheck disable=SC1091
-    virtualenv -p python3 venv && source venv/bin/activate
+    virtualenv -p python3 venv && source venv/bin/activate && install reqs
```

Comment:
so if I use NOENV it won't install anything?

Reply:
yep. I'm thinking of removing that logic too, but thought on doing that on another PR.


Should I clean it up too?

Reply:
Clean up would be something like "always try to create the virtual env and always try to install the requirements"

Reply:
sorry, I missed this comment

> "always try to create the virtual env and always try to install the requirements"

yes, I agree

Reply:
but feel free to do it separatedly

Reply:
I added the changes in the previous patch